src/containers/ConceptPage/components/ConceptMetaData.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { Fragment } from 'react';
+import { injectT, tType } from '@ndla/i18n';
+import { useFormikContext } from 'formik';
+import FormikField from '../../../components/FormikField';
+import { MultiSelectDropdown } from '../../../components/Dropdown';
+import AsyncSearchTags from '../../../components/Dropdown/asyncDropdown/AsyncSearchTags';
+import { FormikMetaImageSearch } from '../../FormikForm';
+
+import { SubjectType, SearchResult } from '../../../interfaces';
+import { ConceptFormValues } from '../conceptInterfaces';
+import InlineImageSearch from './InlineImageSearch';
+
+interface Props {
+  subjects: SubjectType[];
+  fetchTags: (input: string, language: string) => Promise<SearchResult>;
+  inModal: boolean;
+}
+
+const ConceptMetaData = ({ subjects, fetchTags, inModal, t }: Props & tType) => {
+  const { values } = useFormikContext<ConceptFormValues>();
+
+  return (
+    <Fragment>
+      {inModal ? (
+        <InlineImageSearch name="metaImageId" />
+      ) : (
+        <FormikField name="metaImageId">
+          {({ field, form }) => (
+            <FormikMetaImageSearch
+              metaImageId={field.value}
+              setFieldTouched={form.setFieldTouched}
+              showRemoveButton
+              {...field}
+            />
+          )}
+        </FormikField>
+      )}
+      <FormikField name="subjects" label={t('form.subjects.label')}>
+        {({ field }) => (
+          <MultiSelectDropdown labelField="name" minSearchLength={1} data={subjects} {...field} />
+        )}
+      </FormikField>
+      <FormikField
+        name="tags"
+        label={t('form.categories.label')}
+        description={t('form.categories.description')}>
+        {({ field, form }) => (
```

Comment:
Bytte ut med `useFormikContext` ?

Reply:
Egentlig så kunne vi vel klart oss uten noen av de, går jo ann å lage uncontroller felt som du bare sier navnet til.

Grunnen til at dette er slik er mest for å begrense hvor mange endringer jeg gjorde til å bare ta seg av hvor mange props som sendes mellom komponentene.

Reply:
Vi kunne vel også om vi skulle være fancy laget formik felt komponenter som tar ibruk useField internt, så får de alt de trenger med et navn,